conanfile.py

```diff
@@ -15,6 +15,7 @@ class Libnest2DConan(ConanFile):
     topics = ("conan", "cura", "prusaslicer", "nesting", "c++", "bin packaging")
     settings = "os", "compiler", "build_type", "arch"
     build_policy = "missing"
+    generators = "CMakeDeps"
```

Comment:
I would remove this line and keep this generator in the `generate` method. So all generators are clustered together.

Reply:
https://github.com/Ultimaker/libnest2d/pull/8/commits/ca60db19bc503aedec491075282fec1eb0b7882c